### PR TITLE
Ore Box interface update

### PR DESCRIPTION
**Pull request by nightred**
## About The Pull Request
tgui-net the Ore Box, it now updates automatically as you add ores, looks fancy, and includes instructions.

Ore Box:
ore_box

## Why It's Good For The Game
tgui-next is the future

## Changelog
:cl:
add: Ore Box uses tgui-next now
/:cl:

### CODE REVIEW

File: tgui-next/packages/tgui/interfaces/OreBox.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  const { ref } = config;
+  const { materials } = data;
+  return (
+    <Fragment>
+      <Section
+        title="Ore Box"
+      >
+        <BlockQuote mb={1}>
+          If you are wearing a mining stachel on your belt or in a pocket and dragging the box,
+          all ores will be places in here.<br />
```
Comment (Buggy123) [inline tgui-next/packages/tgui/interfaces/OreBox.js]: Should be "placed"

Comment (nightred) [reply tgui-next/packages/tgui/interfaces/OreBox.js]: fixed up

File: tgui-next/packages/tgui/interfaces/OreBox.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  const { state } = props;
+  const { config, data } = state;
+  const { ref } = config;
+  const { materials } = data;
+  return (
+    <Fragment>
+      <Section
+        title="Ore Box"
+      >
+        <BlockQuote mb={1}>
```
Comment (TheVekter) [inline tgui-next/packages/tgui/interfaces/OreBox.js]: Remove the block quote? It looks weird.

File: tgui-next/packages/tgui/interfaces/OreBox.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+          If you are wearing a mining stachel on your belt or in a pocket and dragging the box,
+          all ores will be placed in here.<br />
+          Gibtonite is not accepted.
+        </BlockQuote>
+        <Box>
+          <Box inline color="label" mr={1}>
+            Remove all ores from the box
+          </Box>
+          <Button
```
Comment (TheVekter) [inline tgui-next/packages/tgui/interfaces/OreBox.js]: You can take the <button> contents and put them in the <section> header (under line 14) and it'll put that in the same line as the section title. Give that a shot and see how it looks, I don't really think an explanation for what "Empty" does is necessary and this probably flows better.

File: tgui-next/packages/tgui/interfaces/OreBox.js (hunk 2 above)
Comment (stylemistake) [reply tgui-next/packages/tgui/interfaces/OreBox.js]: Looks fine to me.

**Review by stylemistake — DISMISSED** [review]

File: tgui-next/packages/tgui/interfaces/OreBox.js (hunk 2 above)
Comment (TheVekter) [reply tgui-next/packages/tgui/interfaces/OreBox.js]: Just a suggestion. Maybe it's because of the text under it that's throwing me off.

**Comment by nightred** [comment]
The layout has been tweaked again here is the new look.
orebox2

File: tgui-next/packages/tgui/interfaces/OreBox.js (hunk 3 above)
Comment (nightred) [reply tgui-next/packages/tgui/interfaces/OreBox.js]: This is changed up to look better now.

**Comment by stylemistake** [comment]
Rebase this please on top of master.

**Comment by nightred** [comment]
Did i make it worse or better?

**Comment by TheVekter** [comment]
Oh yeah, that looks nice. Well done.

**Pull request merged**